app/src/screens/CallInvite/index.tsx

```diff
@@ -50,7 +50,8 @@ const styles = StyleSheet.create({
     justifyContent: 'space-around',
   },
   logo: {
-    maxHeight: '25%',
+    height: 50,
+    width: 150,
```

Comment:
Is this the proper aspect ratio for the image? We typically only glue either the height or the width and let the system handle the scaling so the aspect ratio is preserved.

Reply:
Discussed in a call with @kpchoy we're going to leave this as-is for now and file a ticket to use relative measurements everywhere.